src/apps/benchmarks/benchmarkH3Api.c

```diff
@@ -34,4 +34,11 @@ BENCHMARK(h3ToGeoBoundary, 10000, {
     H3_EXPORT(h3ToGeoBoundary)(hex, &outBoundary);
 });
 
+H3Index* out = malloc(H3_EXPORT(maxKringSize)(40) * sizeof(H3Index));
```

Comment:
Looks like we still need to remove these benchmarks from this file, right?

Reply:
This might not have been clear in context, but the added benchmarks were appearing twice, once in `benchmarkH3Api.c` and once in `benchmarkKRing.c` - that's what I was blocking on.